.travis/linux.win32.install.sh

```diff
@@ -27,8 +29,10 @@ sudo apt-get install -y $PACKAGES
 
 # ccache 3.2 is needed because mingw32-x-gcc is version 4.9, which causes cmake
 # to use @file command line passing, which in turn ccache 3.1.9 doesn't support
```

Comment:
Since `TRAVIS_TAG` isn't immediately associated with `ccache`, we should have a comment explaining why `ccache` is disabled for tagged releases.